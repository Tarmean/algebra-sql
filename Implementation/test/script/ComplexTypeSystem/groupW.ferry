for p in table Player (name String, team String) with keys ((name, team))
 group with p.team
return p