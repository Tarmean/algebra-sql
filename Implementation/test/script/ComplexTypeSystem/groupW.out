 => [(String, [{name :: String, team :: String}])]